lesson_08/homework_08.py

```diff
@@ -0,0 +1,26 @@
+list_01 = ["2, 5, 7, 11",
+"abab12, 11, 18, c",
+"1, 12, 33, 14",
+"abetka, 1",
+"1, 2, 3, 4",
+"2, 5, !, ^"]
+
+def sum_of_elements(arg: str):
+    """Calculates the sum of all numbers in the string. If string contains non number raises the exception."""
+    numbers = arg.split(", ")
+    temp_sum = 0
+    for i in numbers:
+        i = int(i)
```

Comment:
```suggestion
        i = int(i)
```
от тут загорнути б у `try\except`

Reply:
Hey! Thank you for the comment, at first I tried to put try/except to the first function above the cycle, it worked just fine, but it'd display additional None in the results, what I did not like, so I moved it to the second function to avoid the issue. But I will try it the way you suggested. Thank you again!